File: go/log_level.go
package cuml4go

type LogLevel int

const (
	Off      LogLevel = 0
	Critical LogLevel = 1
	Error    LogLevel = 2
	Warn     LogLevel = 3
	Info     LogLevel = 4
	Debug    LogLevel = 5
	Trace    LogLevel = 6
)

File: rust/src/log_level.rs
#[derive(Debug, Clone, Copy, PartialEq, Eq)]
pub enum LogLevel {
    Off = 0,
    Critical = 1,
    Error = 2,
    Warn = 3,
    Info = 4,
    Debug = 5,
    Trace = 6,
}
